doc/components.md

```diff
@@ -372,12 +379,16 @@ component (`game.add(parallaxComponent`).
 Also, don't forget to add you images to the `pubspec.yaml` file as assets or they wont be found.
 
 The `Parallax` file contains an extension of the game which adds `loadParallax`, `loadParallaxLayer`
-and `loadParallaxImage` so that it automatically uses your game's image cache instead of the global
+, `loadParallaxImage` and `loadParallaxAnimation` so that it automatically uses your game's image cache instead of the global
```

Comment:
nit: can we put the comma on the previous line, it's weird to start a line with a comma

Reply:
Unfortunately not, the previous line is on the exact 100 characters limit.